packages/x-date-pickers/src/internals/hooks/useField/useFieldV6TextField.ts

```diff
@@ -202,6 +202,9 @@ export const useFieldV6TextField: UseFieldTextField<false> = (params) => {
         return nextSectionIndex === -1 ? sections.length - 1 : nextSectionIndex - 1;
       },
       focusField: (newSelectedSection = 0) => {
+        if (getActiveElement(document) === inputRef.current) {
```

Comment:
Avoid resetting/jumping focus to the first section when switching fields with the mouse on a multi input v6 (native input) range field.


https://github.com/user-attachments/assets/98a6eea6-06fa-413d-9d5b-b835b7311179